pom.xml

```diff
@@ -47,5 +47,11 @@
             <artifactId>service</artifactId>
             <version>0.12.0</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
```

Comment:
Is the comment needed? We can search the web for the package if needed.

Reply:
I like to leave the link where I got it